cmd/cerbos/compile/compile.go

```diff
@@ -54,7 +54,7 @@ cerbos compile --skip-tests /path/to/policy/repo
 type Cmd struct { //nolint:govet // Kong prints fields in order, so we don't want to reorder fields to save bytes.
 	Dir           string                            `help:"Policy directory" arg:"" required:"" type:"path"`
 	IgnoreSchemas bool                              `help:"Ignore schemas during compilation"`
-	Tests         string                            `help:"Path to the directory containing tests. Defaults to policy directory." type:"path"`
+	Tests         string                            `help:"[Deprecated] Path to the directory containing tests. Defaults to policy directory." type:"path"`
```

Comment:
We need to update the docs which still imply this is required: https://docs.cerbos.dev/cerbos/latest/policies/compile#_running_tests